### PR TITLE
Updated controller tests to use test containers

### PR DESCRIPTION
**Pull request by bhatiam12**

### CODE REVIEW

File: server/src/test/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteControllerTest.java [hunk 1]
```diff
@@ -1,81 +1,67 @@
 package com.objectcomputing.checkins.services.checkinnotes;
```
Comment (chrismckiernan) [inline server/src/test/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteControllerTest.java:2]: Can you run test coverage on this one file and check the code coverage.

Can you add more tests such as 

- creating an already existing checkin note (checkInId and memberProfileId match).
- trying to create a checkin note with a non existing checkin
- trying to create a checkin note with a non existing memberProfile
- A find with expected multiple results return along with a few notes created that won't be chosen, which should not be expected to be in the results.

Comment (chrismckiernan) [reply server/src/test/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteControllerTest.java:2]: Code looks good.

Comment (bhatiam12) [reply server/src/test/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteControllerTest.java:2]: Sure, I can add more tests. I just converted existing tests to use test containers as we have service tests(mock) as well which will take care of other scenarios.

Comment (chrismckiernan) [reply server/src/test/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteControllerTest.java:2]: Good point...

@mkimberlin Did we decide on whether to abandon the unit tests for impl, and focus all on testcontainers?

Comment (bhatiam12) [reply server/src/test/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteControllerTest.java:2]: done. added more tests and it should cover everything.

**Pull request merged**